.github/workflows/merged.yml

```
@@ -0,0 +1,37 @@
+    steps:
+      - name: 使用 NX 缓存
+        uses: actions/cache/restore@v4
+        with:
+          path: .nx
+          key: NX-${{ runner.os }}-${{ matrix.node-version }}-${{ github.ref_name }}-${{ github.sha }}
+
+      - name: 更新 NX 缓存
+        uses: actions/cache/save@v4
+        with:
+          path: .nx
+          key: NX-${{ runner.os }}-${{ matrix.node-version }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)